src/Components/Search.js

```diff
@@ -0,0 +1,56 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import MediaCard from "./MediaCard";
+import SearchBar from "./SearchBar";
+
+export default function Search(props) {
+
+  const [searchResults, setSearchResults] = useState([])
+
+  const search = async (value) => {
+    let data = await axios.get(`https://images-api.nasa.gov/search?q=${value}`)
+    setSearchResults(data.data.collection.items)
+  }
+  const [saved, setSaved] = useState({})
+
+  const fetchSaved = async function() {
+    const savedFavs = await axios.get(`http://localhost:3001/saved`)
+    setSaved(savedFavs.data)
+  }
+  
+  useEffect(() => {
+    fetchSaved()
+  }, [])
+
+  const arrangeData = (obj) => {
+    return {
+      title: obj.data[0].title,
+      url: obj.links ? obj.links[0].href : null,
+      description: obj.data[0].description,
+      media_type: obj.links ? obj.links[0].render : null,
+      nasaId: obj.data[0].nasa_id
+    }
+  }
+
+  const save = async toSave => {
+    const newData = await axios.post(`http://localhost:3001/image`, toSave)
+    props.snackbar('saved')
+    setSaved(newData.data.saved)
+  }
+
+  const remove = async (id, nasaId) => {
+    const newData = await axios.delete(`http://localhost:3001/image/${id}`,{data: { nasaId }})
+    props.snackbar('removed')
+    setSaved(newData.data.saved)
+  }
```

Comment:
I'm not sure why you need `save` and `remove` methods both in `MediaCard` and `Search`